src/System.Memory/src/System/ThrowHelper.cs

```diff
@@ -120,6 +120,7 @@ internal enum ExceptionArgument
         array,
         culture,
         manager,
-        count
+        count,
+        writer
```

Comment:
tip: adding a trailing `,` to this last line will prevent the next addition from having to edit your line (as this diff had to).